django/contrib/admin/views/main.py

```diff
@@ -174,9 +177,25 @@ def get_filters(self, request):
         may_have_duplicates = False
         has_active_filters = False
 
+        lookup_allowed_supports_request = func_supports_parameter(
```

Comment:
```suggestion
        # RemovedInDjango60Warning.
        lookup_allowed_supports_request = func_supports_parameter(
```

I've also never deprecated a feature but from looking at the codebase it seems as though all extra temp code added during the deprecation period needs to be commented with the `DeprecatedInDjangoXXWarning` like so.

@felixxm & @nessita is this correct? 🤔

Reply:
Yes, they make it easier to remove when deprecation ends. I will try to review it next week.

Reply:
Added, I will also try and remember to make an update to the [docs](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#deprecating-a-feature) :+1: